tox.ini

```diff
@@ -10,6 +10,9 @@ setenv =
   LANGUAGE=en_US
   LC_ALL=en_US.utf-8
   SETUPTOOLS_ENABLE_FEATURES=legacy-editable
+passenv=
+  GITHUB_REF_NAME,
```

Comment:
```suggestion
  GITHUB_REF_NAME
```

Reply:
no commas needed?

Reply:
The commas break things

Reply:
ok. FWIW I followed what i did in qiskit_ibm_runtime and there are commas there

Reply:
It looks like Runtime is fine:

https://github.com/Qiskit/qiskit-ibm-runtime/blob/a2a803e37c08746fe5c02138bd6896414ae1e7a9/tox.ini#L13-L15

FYI you would use commas if they were all on the same line. (INI sucks - not a very good config format)